WalletWasabi.Fluent/Icons/Icons.axaml

```diff
@@ -206,6 +206,14 @@
       <StreamGeometry x:Key="coordinator">M9.25 6C8.83579 6 8.5 6.33579 8.5 6.75C8.5 7.16421 8.83579 7.5 9.25 7.5H14.75C15.1642 7.5 15.5 7.16421 15.5 6.75C15.5 6.33579 15.1642 6 14.75 6H9.25ZM8.5 17.75C8.5 17.3358 8.83579 17 9.25 17H14.75C15.1642 17 15.5 17.3358 15.5 17.75C15.5 18.1642 15.1642 18.5 14.75 18.5H9.25C8.83579 18.5 8.5 18.1642 8.5 17.75ZM9.25 14C8.83579 14 8.5 14.3358 8.5 14.75C8.5 15.1642 8.83579 15.5 9.25 15.5H14.75C15.1642 15.5 15.5 15.1642 15.5 14.75C15.5 14.3358 15.1642 14 14.75 14H9.25ZM6 5C6 3.34315 7.34315 2 9 2H15C16.6569 2 18 3.34315 18 5V19C18 20.6569 16.6569 22 15 22H9C7.34315 22 6 20.6569 6 19V5ZM9 3.5C8.17157 3.5 7.5 4.17157 7.5 5V19C7.5 19.8284 8.17157 20.5 9 20.5H15C15.8284 20.5 16.5 19.8284 16.5 19V5C16.5 4.17157 15.8284 3.5 15 3.5H9Z</StreamGeometry>
+      <StreamGeometry x:Key="manual_control">F1 M3.88411 4.01327C3.74935 3.85155 3.52773 3.79156 3.32979 3.86322C3.13185 3.93489 3 4.12285 3 4.33336L3 8.66667C3 8.94287 3.22386 9.16667 3.5 9.16667C3.77614 9.16667 4 8.94287 4 8.66667L4 5.7144L5.61587 7.65347C5.71087 7.76747 5.8516 7.83333 6 7.83333C6.1484 7.83333 6.28913 7.76747 6.38413 7.65347L8 5.7144L8 8.66667C8 8.94287 8.22387 9.16667 8.5 9.16667C8.77613 9.16667 9 8.94287 9 8.66667L9 4.33336C9 4.12285 8.86813 3.93489 8.6702 3.86322C8.47227 3.79156 8.25067 3.85155 8.11587 4.01327L6 6.55233L3.88411 4.01327ZM2.16667 0.5C0.970047 0.5 0 1.47005 0 2.66667L0 10.3333C0 11.5299 0.970047 12.5 2.16667 12.5L9.83333 12.5C11.0299 12.5 12 11.5299 12 10.3333L12 2.66667C12 1.47005 11.0299 0.5 9.83333 0.5L2.16667 0.5ZM1 2.66667C1 2.02233 1.52233 1.5 2.16667 1.5L9.83333 1.5C10.4777 1.5 11 2.02233 11 2.66667L11 10.3333C11 10.9777 10.4777 11.5 9.83333 11.5L2.16667 11.5C1.52233 11.5 1 10.9777 1 10.3333L1 2.66667Z</StreamGeometry>
+
+      <DrawingImage x:Key="automatic_control">
+        <DrawingGroup>
+          <GeometryDrawing Brush="{DynamicResource TextForegroundColor}" Geometry="F1 M2.16667 0.5C0.973333 0.5 0 1.47333 0 2.66667L0 10.3333C0 11.5267 0.973333 12.5 2.16667 12.5L9.83333 12.5C11.0267 12.5 12 11.5267 12 10.3333L12 2.66667C12 1.47333 11.0267 0.5 9.83333 0.5L2.16667 0.5ZM1 2.66667C1 2.02 1.52 1.5 2.16667 1.5L9.83333 1.5C10.48 1.5 11 2.02 11 2.66667L11 10.3333C11 10.98 10.48 11.5 9.83333 11.5L2.16667 11.5C1.52 11.5 1 10.98 1 10.3333L1 2.66667Z" />
+          <GeometryDrawing Brush="{DynamicResource TextForegroundColor}" Geometry="F1 M8.78696 8.44673L6.45362 3.61339C6.28696 3.26673 5.72029 3.26673 5.55362 3.61339L3.22029 8.44673C3.10029 8.69339 3.20696 8.99339 3.45362 9.11339C3.70696 9.23339 4.00029 9.12673 4.12029 8.88006L4.54696 7.99339L7.46029 7.99339L7.88696 8.88006C7.97362 9.06006 8.15362 9.16006 8.34029 9.16006C8.41362 9.16006 8.48696 9.14673 8.56029 9.11339C8.80696 8.99339 8.91362 8.69339 8.79362 8.44673L8.78696 8.44673ZM5.02696 7.00006L6.00029 4.98673L6.97362 7.00006L5.02696 7.00006Z" />
+        </DrawingGroup>
+      </DrawingImage>
```

Comment:
If you export the icon as SVG from figma, you can get the path from it, use only a `StreamGeometry` please.

```
M3.25 0C1.46 0 0 1.46 0 3.25V14.75C0 16.54 1.46 18 3.25 18H14.75C16.54 18 18 16.54 18 14.75V3.25C18 1.46 16.54 0 14.75 0H3.25ZM1.5 3.25C1.5 2.28 2.28 1.5 3.25 1.5H14.75C15.72 1.5 16.5 2.28 16.5 3.25V14.75C16.5 15.72 15.72 16.5 14.75 16.5H3.25C2.28 16.5 1.5 15.72 1.5 14.75V3.25Z M13.1799 11.92L9.67995 4.67003C9.42995 4.15003 8.57995 4.15003 8.32995 4.67003L4.82995 11.92C4.64995 12.29 4.80995 12.74 5.17995 12.92C5.55995 13.1 5.99995 12.94 6.17995 12.57L6.81995 11.24H11.1899L11.8299 12.57C11.9599 12.84 12.2299 12.99 12.5099 12.99C12.6199 12.99 12.7299 12.97 12.8399 12.92C13.2099 12.74 13.3699 12.29 13.1899 11.92H13.1799ZM7.53995 9.75003L8.99995 6.73003L10.4599 9.75003H7.53995Z
```

Reply:
Fixed.